src/main/java/com/fundynamic/d2tm/Game.java

```diff
@@ -27,6 +27,7 @@ public void render(GameContainer container, Graphics g) throws SlickException {
     @Override
     public void init(GameContainer gameContainer) throws SlickException {
         try {
+            gameContainer.setVSync(true);
```

Comment:
Hrm, just a thought: I'd expect this setting to be closer to screen resolution initialization, since they're both graphics settings.